controllers/discovery/searchUser.js

```diff
@@ -101,6 +102,7 @@ const SearchUser = async (req, res) => {
           allow_anon_dm,
           userId,
           minimumKarmaScore: MINIMUM_KARMA_SCORE,
+          admin_user_id: process.env.BETTER_ADMIN_ID,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This line introduces a potential security risk. Directly using environment variables in your code can expose sensitive information. It's better to use a configuration file or module to manage environment variables. This way, you can validate and sanitize the values before using them in your application.
```diff
- admin_user_id: process.env.BETTER_ADMIN_ID,
+ admin_user_id: config.get('BETTER_ADMIN_ID'),
```
In the above diff, `config` is a hypothetical module that you would create to handle your configuration.


<!-- This is an auto-generated comment by OSS CodeRabbit -->